VideoPlayer.js

```diff
@@ -693,7 +695,19 @@ export default class VideoPlayer extends Component {
         this.setVolumePosition( position );
         state.volumeOffset = position;
         this.mounted = true;
-
+        this.setState({uri: this.props.source.uri});
+        this.loadAnimation(); // this one does not go
+        if(this.props.source && this.props.source.uri && this.props.source.uri.includes('theplatform.com')) {
```

Comment:
if `this.props.source` is null or undefined this method would crash on line 698, so it doesn't add anything to check on line 700. Please either add a check on line 698 or remove the one on 700.

Reply:
Solved